rtv/page.py

```diff
@@ -251,6 +252,70 @@ def login(self):
         else:
             show_notification(self.stdscr, ['Welcome {}'.format(username)])
 
+    @BaseController.register('d')
+    def delete(self):
+        """
+        Delete a submission or comment.
+        """
+        data = self.content.get(self.nav.absolute_index)
+        if data['author'] != self.reddit.user.name:
+            show_notification(self.stdscr, ['You can\'t delete this'])
+            return
+
+        prompt = 'Are you sure you want to delete this? (y/n):'
+        char = self.prompt_input(prompt)
+        if char != 'y':
+            show_notification(self.stdscr, ['Delete cancelled'])
+            return
+
+        try:
+            data['object'].delete()
+            show_notification(self.stdscr, ['Deleted'])
+        except praw.errors.APIException as e:
+            show_notification(self.stdscr, [e.message])
+        else:
+            time.sleep(0.5)
+            self.refresh_content()
+
+    @BaseController.register('e')
+    def edit(self):
+        """
+        Edit a submission or comment.
+        """
+        data = self.content.get(self.nav.absolute_index)
+        if data['author'] != self.reddit.user.name:
+            show_notification(self.stdscr, ['You can\'t edit this'])
+            return
+
+        if data['type'] == 'Submission':
+            subreddit = self.reddit.get_subreddit(self.content.name)
+            sub = str(subreddit).split('/')[2]
+            sub_file = '{content}' + SUBMISSION_FILE
```

Comment:
I think it would be better to have this be the default ``SUBMISSION_FILE`` template, and set ``content=''`` in the other case.